rules/eslint/style/walmart.js

```diff
@@ -26,6 +28,8 @@ module.exports = {
     "id-length": 0,
     // this option sets a specific tab width for your code
     "indent": [2, 2],
+    // specify whether double or single quotes should be used in JSX attributes
+    "jsx-quotes": 0,
```

Comment:
Personally, I like `[1, "double"]` as the configuration there, but it'd probably be good to have someone like @kenwheeler weigh in on his preference (since he's writing most of it).